src/TestExplorer/TestExplorer.ts

```diff
@@ -147,18 +149,27 @@ export class TestExplorer {
             if (isPathInsidePath(uri.fsPath, folder.folder.fsPath)) {
                 const target = folder.swiftPackage.getTarget(uri.fsPath);
                 if (target && target.type === "test") {
```

Comment:
I guess for swift-testing we will need to remove this condition given `swift-testing` tests can be found in any target.

Reply:
At this time, we would only want to support running tests in test targets. We document that requirement [here](https://swiftpackageindex.com/apple/swift-testing/0.7.0/documentation/testing/definingtests#app-top) (see the first **Note** callout.)